### ISSUE TITLE
How do I montor for heart rate, like we do for steps ?

### ISSUE THREAD

**Issue by theapache64** [Fitpolo/FitpoloDemo_H706_Android#2]

**Comment by liuwenzheng** [comment]
Heart rate is automatically measured, and you can set the time interval to measure your heart rate:
17.set heart rate  meansure intervial
	ZWriteHeartRateIntervalTask
	 Incoming parameters need pass heart rate intervial
	heartRateInterval;// 0： off；1： 10mins；2： 20mins；3： 30mins

**Comment by theapache64** [comment]
Yeah, I got it from the docs. but what I want is a callback which will be invoked when ever a new heart rate is added in the band. 

How can i do that ?

For example, there a task named `ZOpenStepListenerTask` which can be used to monitor on step change. Like that, I need to monitor for heart rate.

**Comment by liuwenzheng** [comment]
Sorry, we don't have that yet

**Comment by theapache64** [comment]
Yeah, I haven't see any method in the SDK. Is it possible to do ? What are the bytes to be passed and will be received ?

**Comment by liuwenzheng** [comment]
The firmware for the device does not support it
